test/gtest/uct/test_p2p_am.cc

```diff
@@ -489,4 +493,4 @@ UCS_TEST_P(uct_p2p_am_misc, am_max_short_multi) {
     EXPECT_EQ(UCS_OK, status);
 }
 
-UCT_INSTANTIATE_TEST_CASE(uct_p2p_am_misc)
+UCT_INSTANTIATE_NO_SELF_TEST_CASE(uct_p2p_am_misc)
```

Comment:
@yosefe Where are two tests in this case.
"no_rx_buffs" (mentioned above) has no loopback support.
"am_max_short_multi" has a loop to call am_short until exhaust all resources https://github.com/openucx/ucx/blob/master/test/gtest/uct/test_p2p_am.cc#L479

This test is hungs because loopback doesn't require additional resources from call to call in case of callback_am returns UCS_OK. It reuses the buffer.